augur/align.py

```diff
@@ -56,14 +57,16 @@ def run(args):
     # align
     if args.method=='mafft':
         cmd = "mafft --reorder --anysymbol --thread %d %s 1> %s 2> %s.log"%(args.nthreads, seq_fname, output, output)
-        os.system(cmd)
         print("\nusing mafft to align via:\n\t" + cmd +
               " \n\n\tKatoh et al, Nucleic Acid Research, vol 30, issue 14"
               "\n\thttps://doi.org/10.1093%2Fnar%2Fgkf436\n")
     else:
         print('ERROR: alignment method not implemented')
         return -1
 
+    if not run_shell_command(cmd):
```

Comment:
This reads as overly opaque to me. It's not obvious to me that this actually does run `cmd`. Is there a clearer way this could be written?

Reply:
maybe change to 
```
success = run_shell_command(cmd)
if not success:
...
```

Reply:
I agree with @trvrb and second @rneher's suggestion, or similar.